### PR TITLE
Add skeleton for product support SLE Micro OS with name slmicro5

### PR DESCRIPTION
**Pull request by teacup-on-rockingchair**
#### Description:

- Add support for SUSE Linux Enterprise MicroOS product with target name slmicro5

#### Rationale:

- Add required definitions to support slmicro5 product
- The name was chosen with the idea that although it is a SLE(SUSE Linux Enterprise) product it has different behaviour than sle12 and sle15 platforms and should be considered part of new multi_platform slmicro, designated for all SUSE Linux Enterprise MicroOS products.
##### Internals
- Add skeleton for product support SLE Micro OS
- Add manual DISA STIG for slmicro5 as reference
- Add stig control file for DISA STIG profile
- Add rule installed_OS_is_vendor_supported

### CODE REVIEW

File: linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/rule.yml [hunk 1]
```diff
@@ -34,6 +34,7 @@ identifiers:
     cce@rhel9: CCE-83453-1
     cce@sle12: CCE-83001-8
     cce@sle15: CCE-83260-0
+    cce@slmicro5: CCE-93601-3
```
Comment (jan-cerny) [inline linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/rule.yml:37]: Will there be a CCE pool file, similar to `shared/references/cce-sle15-avail.txt`?

File: products/slmicro5/profiles/stig.profile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+documentation_complete: true
+
+metadata:
+    version: V1R1
+    SMEs:
+        - teacup-on-rockingchair
+
+reference: https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux
+
+title: 'SUSE Linux Enterprise Micro (SLEM) 5'
```
Comment (jan-cerny) [inline products/slmicro5/profiles/stig.profile:10]: There should be STIG in the profile title

**Review by jan-cerny — COMMENTED** [review]
so far looks great

File: linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/rule.yml (hunk 1 above)
Comment (teacup-on-rockingchair) [reply linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/rule.yml:37]: For sure there is a pending PR #12109 on me for that, that I will review/merge as soon as the current one is approved

File: products/slmicro5/profiles/stig.profile (hunk 2 above)
Comment (teacup-on-rockingchair) [reply products/slmicro5/profiles/stig.profile]: Thanks added in [96606f8](https://github.com/ComplianceAsCode/content/pull/12028/commits/96606f82dc34b957a718811160de0abed1e87514)

**Review by jan-cerny — APPROVED** [review]
I have built the slmicro5 product. I have seen the built data stream. I have also read the generated HTML guide for STIG.

**Comment by jan-cerny** [comment]
The CI fail isn't caused by the contents of this PR. The Code Climate problem is a false positive.

**Pull request merged**